architecture/aws/observability.md

```diff
@@ -5,61 +5,62 @@ reviewed: 2024-03-14
 
 [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) is a monitoring and observability service that allows one to collect and access performance and operational data in the form of logs and metrics on a single platform.
 
 :heavy_plus_sign: Pros:
 
-- A single tool to visualize metrics and logs emitted by the system
+- Single tool to visualize metrics and logs emitted by the system
 - Fully managed service
-- Tightly integrated with other AWS services, such as AWS Lambda and Amazon EC2
-- Supports querying data from multiple s&#111;urces, allowing one to monitor metrics on AWS, on premises or other clouds
-- Customizable dashboards and automated alarms and actions
+- Tightly integrated with other AWS services
+- Supports querying data from multiple sources
+- Customizable dashboards with automated alarms and remediating actions
 - Real-time telemetry
 
 :heavy_minus_sign: Cons:
 
-- Amazon CloudWatch does not support traces
-- AWS CloudWatch offers a [free tier](https://aws.amazon.com/cloudwatch/pricing/), but costs may escalate with increases usage, requiring dedicated monitoring
+- Does not support traces
+- [Costs](https://aws.amazon.com/cloudwatch/pricing/) can escalate with increased usage
 
 The Particular Service Platform collects metrics in two forms:
 
-- OpenTelemetry-based metrics, which can be collected by enabling OpenTelemetry and exporting the metrics to Amazon CloudWatch
-- Custom metrics with NServiceBus.Metrics which can be exported to Amazon CloudWatch.
+1. OpenTelemetry-based metrics - can be collected by [enabling OpenTelemetry](/nservicebus/operations/opentelemetry.md) and exporting the metrics to Amazon CloudWatch.
+1. Custom metrics with `NServiceBus.Metrics` - can be exported to Amazon CloudWatch - TODO: Link this to TODO2 below.
 
-Try the Amazon CloudWatch sample for NServiceBus metrics and logs →
+Try the Amazon CloudWatch sample for OpenTelemetry → TODO: Link this to TODO1 below.
 
 **TODO**:
 
-1. Create a sample that uses NSB metrics, which integrates with Cloudwatch
+1. Create a sample that uses Otel and integrates with CloudWatch
+1. Create a sample that uses NSB metrics, which integrates with CloudWatch
     1. Collect NSB metrics, send to cloudwatch
     2. Collect [NLog logs with extensions.logging](/samples/logging/extensions-logging/), [send to Cloudwatch](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/configure-logging-for-net-applications-in-amazon-cloudwatch-logs-by-using-nlog.html)
 
-#### Amazon X-Ray
+## Amazon X-Ray
 
-[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can collect trace data from the applications, providing insights that can help identify issues or bottlenecks that could benefit from optimization.
+[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can receive trace data from applications. The trace data can provide insights that can help identify performance bottlenecks, edge case errors, and other hard to detect issues.
```

Comment:
```suggestion
[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can receive trace data from applications. The trace data can provide insights to help identify the root cause of failures, hard-to-detect issues or side effects, and performance bottlenecks.
```

I think changing the order is important because it's primarily built to help do root-cause analysis of failures or weird side effects and, secondarily, performance bottlenecks.
If you look at the use cases on the [xray page](https://aws.amazon.com/xray/), analyze and debug are first, performance analytics later.